src/Models/Behaviors/HasNesting.php

```diff
@@ -35,7 +35,7 @@ public function getNestedSlugAttribute()
      */
     public function getAncestorsSlug(?string $locale = null): string
     {
-        return $this->getAncestors()->reverse()
+        return $this->getAncestors()->sortBy('_lft')
```

Comment:
```suggestion
        return $this->getAncestors()->sortBy($this->getLftName())
```

Considering that our generated migration relies on [the package helper](https://github.com/area17/twill/blob/3a01156078c1fed7c10523b8c8c0cd765b18cd9a/src/Commands/stubs/migration.stub#L28C21-L28C30), I think we should also rely on [the package method to retrieve the name of that column](https://github.com/lazychaser/laravel-nestedset/blob/f7a36eb3779ee4119a8ea72404aaf90c44fc9a15/src/NodeTrait.php#L845-L848).

Reply:
Updated